wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -213,6 +215,41 @@ public function render_media( $instance ) {
 	}
 
 	/**
+	 * Filters the default media display states for items in the Media list table.
+	 *
+	 * @since 4.8.0
+	 * @access public
+	 *
+	 * @param array   $states An array of media states.
+	 * @param WP_Post $post   The current attachment object.
+	 * @return array
+	 */
+	public function display_media_state( $states, $post ) {
```

Comment:
Should this maybe be moved to the base `WP_Widget_Media` class? Naturally some of the hard-coded values here would need to change, like replace `'image/'` with `$this->widget_options['mime_type']` and use `$this->name` instead of `'Image Widget'`.